api/webscraper/nyserda_scraper.py

```diff
@@ -1,93 +1,175 @@
-  else:
-    large_data = nyserda_large_response.json()
-    filtered_list = []
-    for item in large_data:
-      if check_status(item.get('project_status', None)) != 'Cancelled':
-        project_dict = {'project_name': item.get('project_name', None),
-                        'project_status': check_status(item.get('project_status', None)), 
-                        'renewable_energy_technology': item.get('renewable_technology', None),
-                        'size': item.get('new_renewable_capacity_mw', None),
-                        'developer': item.get('developer_name', None),
-                        'proposed_cod': item.get('year_of_delivery_start_date', None),
-                        'county': item.get('county_province', None),
-                        'region': item.get('redc', None),
-                        'zipcode': item.get('zip_code', None),
-                        'latitude': item.get('georeference')['coordinates'][1] if item.get('georeference', None) is not None else None,
-                        'longitude': item.get('georeference')['coordinates'][0] if item.get('georeference', None) is not None else None,
-                        # 'data_through_date': item.get('data_through_date', None),
-                        'interconnection_queue_number': item.get('interconnection_queue_number', None),
-                        'key_development_milestones': None,
-                        'project_image': None,
-                        'approved': False
-        }
-        filtered_list.append(project_dict)
-    return filtered_list
+    nyserda_large_response = requests.get("https://data.ny.gov/resource/dprp-55ye.json")
+    if nyserda_large_response.status_code != 200:
+        raise ValueError(
+            "Request to NYSERDA failed. Status code: %d\n%s"
+            % (nyserda_large_response.status_code, nyserda_large_response.text)
+        )
+    else:
+        large_data = nyserda_large_response.json()
+        filtered_list = []
+        for item in large_data:
+            item["renewable_technology"] = (
+                standardize_label(item.get("renewable_technology"))
+                if item.get("renewable_technology", None) is not None
+                else None
+            )
+            if (
+                check_status(item.get("project_status", None)) != "Cancelled"
+                and item.get("renewable_technology", None) in renewable_energy_set
+            ):
+                project_dict = {
+                    "project_name": item.get("project_name", None),
+                    "project_status": check_status(item.get("project_status", None)),
+                    "renewable_energy_technology": item.get(
+                        "renewable_technology", None
+                    ),
+                    "developer": item.get("developer_name", None),
+                    "proposed_cod": item.get("year_of_delivery_start_date", None),
```

Comment:
don't get proposed COD from year_of_delivery_start_date. Only NYISO has the proposed COD

Reply:
@itsliterallymonique would you like me to remove the inclusion of the `proposed_cod` date from the NYSERDA small-scale solar projects as well? currently, we're using the "interconnection_date" field for the small-scale solar projects as an estimate for the proposed COD

Reply:
@deenasun hmmm. Let's keep that for now in the small-scale solar database. I will check with our POC if we can use interconnection_date for proposed_cod. I am not too sure if they are the same.

Reply:
Done!